releasenotes/notes/civisibilitywriter-3fa0ac9af42256c2.yaml

```diff
@@ -0,0 +1,3 @@
+features:
+  - |
+    tracing: Introduces ``CIVisibilityWriter``, an ``HTTPWriter`` subclass that communicates with Datadog's CI Visibility intake endpoint
```

Comment:
```suggestion
    tracing: Introduces ``CIVisibilityWriter``, an ``HTTPWriter`` subclass that communicates with Datadog's CI Visibility intake endpoint. This will be automatically configured for the pytest plugin.
```

Reply:
Would it be fair to say "An upcoming change will configure the pytest plugin to automatically use this writer."?

Reply:
Yup, that sounds good to me.